roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,12 @@
 dependencies:
-  - pip
+  - role: python-app
+    name: zuul
+    python_pipdeps:
+      - name: jenkins-job-builder
+        version: '1.6.1'
+      - name: pyzmq
+      - name: ansible
+      - name: statsd
+    git_repo: "{{ zuul_git_repo_url }}"
+    git_version: "{{ zuul_git_branch }}"
+    python_notify: Restart zuul
```

Comment:
Are you sure this works? You'll be calling the python-app role before the zuul role, and the Restart zuul handler lives in the zuul role. I don't think you'll be able to call it from the python-app role. Please test.